docs/release_notes.md

```diff
@@ -6,20 +6,24 @@
 
 - Attachment to geometry constraints in AdnMuscle and AdnRibbonMuscle solvers. Multiple target geometries are supported.
 - Slide on geometry constraints in AdnMuscle and AdnRibbonMuscle solvers. Multiple target geometries are supported.
+- Shape preservation constraints integrated in all solvers.
 - Point mass support integrated in all solvers to control the dynamics of the system.
+- Compute point mass distribution based on the density and volume or area.
 
 ### Maya
 
-- Added AdnSkinMerge deformer to blend the results from animated and simualted input geometries.
+- Added AdnSkinMerge deformer to blend the results from animated and simulated input geometries.
 - Added support for attachment to geometry in AdnMuscle and AdnRibbonMuscle solvers.
 - Added support to slide on geometry in AdnMuscle and AdnRibbonMuscle solvers.
+- Added support for shape preservation in all AdonisFX solvers.
```

Comment:
What is the difference between this line and the one above (line 9)? If it is about debugging, maybe is worth moving it to the line about it.

Reply:
Added shape preservation as a new debuggable feature in line 23.

Reply:
Here we are not referring to the constraints but rather the system as a whole. I would not say "all solvers" as in the future we will maybe have more deformers/solvers and it might be difficult to track it back. Can we be more verbose? List the deformers please